prototype2/gdgem/nmx/ReadoutTest.cpp

```diff
@@ -0,0 +1,68 @@
+/** Copyright (C) 2016, 2017 European Spallation Source ERIC */
+
+#include <gdgem/nmx/Readout.h>
+#include <test/TestBase.h>
+
+class NMXReadoutTest : public TestBase {
+protected:
+  virtual void SetUp() {
+    if (boost::filesystem::exists("readout_file_test.h5"))
```

Comment:
Would it be better to use the in-memory file option?

Reply:
It this case, maybe you are right. In other cases we also test opening a file for reading so it has to persist.

Reply:
I think it is best to avoid disk io in unit tests. You have separate test runners in these files though, so I guess you intend them to be run separately anyway?